source/type/data.cpp

```diff
@@ -484,6 +489,26 @@ void Data::aggregate_odt() {
     }
 }
 
+void Data::fill_stop_point_address() {
+    size_t without_address = 0;
+    for (auto sp : pt_data->stop_points) {
```

Comment:
```suggestion
    for (const auto& sp : pt_data->stop_points) {
```
?

Reply:
No I modify it, [see](https://github.com/CanalTP/navitia/pull/3434/files#diff-b18f3fcd9eef06ffc9132fca1d60bdfcf7c92bf0bd36f2fb29a554b679e2d2f8R501)